cirq-google/cirq_google/engine/qcs_notebook.py

```diff
@@ -96,17 +100,18 @@ def get_qcs_objects_for_notebook(
             print(f"Available processors: {[p.processor_id for p in processors]}")
             print(f"Using processor: {processor.processor_id}")
         if not project_id:
-            project_id = processor.project_id
+            project_id = getattr(processor, 'project_id', getattr(processor, '_project_name', None))
```

Comment:
Should this instead use `is_virtual` to determine the default project_id?

Reply:
Added this to AbstractLocalProcessor instead.